change/@microsoft-fast-foundation-12c0b3c8-515f-4b6f-a47b-0d6124919595.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Import icons for fast-foundation stories",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "47367562+bheston@users.noreply.github.com",
+  "dependentChangeType": "prerelease"
```

These only affect the stories and other dev environment changes, so both of the "type" and "dependentChangeType" can be "none".
```suggestion
  "type": "none",
  "comment": "Import icons for fast-foundation stories",
  "packageName": "@microsoft/fast-foundation",
  "email": "47367562+bheston@users.noreply.github.com",
  "dependentChangeType": "none"
```